### ISSUE TITLE
AppImage fails on linux with symbol lookup error

### ISSUE THREAD

**Issue by solsticedhiver** [cjcliffe/CubicSDR#697]
On ubuntu 18.10, trying to run the appimage gives the following error:

```
[ERROR] SoapySDR::loadModule(.//////////lib/SoapySDR/modules0.7/libsdrPlaySupport.so)
  dlopen() failed: libmirsdrapi-rsp.so.2.13: cannot open shared object file: No such file or directory
	Available factories...airspy, airspyhf, audio, bladerf, hackrf, lime, null, redpitaya, remote, rtlsdr
./CubicSDR-0.2.5-x86_64.AppImage: symbol lookup error: /usr/lib/x86_64-linux-gnu/libGLX_mesa.so.0: undefined symbol: xcb_dri3_get_supported_modifiers
```
One gets briefly the GUI appearing then closing.

That's 2 errors:
- one for the non existent libmirsdrapi-rsp.so lib.
- one for the symbol lookup error in libGLX_msea.so.

**Comment by cjcliffe** [comment]
@solsticedhiver this may be a known issue from what I've quickly googled.

Does it work if you run it like this?

`LD_PRELOAD=/usr/lib/x86_64-linux-gnu/libxcb-dri3.so.0 ./CubicSDR-0.2.5-x86_64.AppImage`

**Comment by solsticedhiver** [comment]
@cjcliffe yes that works. thanks

**Comment by mjpowersjr** [comment]
I can confirm this works for me using Ubuntu 19.10 as well.

**Comment by RepComm** [comment]
Can confirm fix working using Ubuntu 20.04LTS

**Comment by Abrahammar997** [comment]
Don't work on Ubuntu 21.04. It runs but when "snips" it make window black
